File: chapter 1/ex_1_3.php
function solution($a, $b, $c) {
  if ($a < $b && $a < $c) {
    return ($b ** 2) + ($c ** 2);
  } elseif ($b < $a && $b < $c) {
    return ($a ** 2) + ($c ** 2);
  }
  return ($a ** 2) + ($b ** 2);
}

echo solution(3, 2, 1);

File: chapter 1/ex_1_3.py
def solution(a, b, c):
    if a < b and a < c:
        return b** 2 + c ** 2
    if b < a and b < c:
        return a ** 2 + c ** 2
    else:
        return a ** 2 + b ** 2

print(solution(3, 2, 1))
